versions/next/asyncapi.md

```diff
@@ -66,22 +67,25 @@ It means [processes](#definitionsProcess) can subscribe to `user/signedup` chann
 ## Definitions
 
 #### <a name="definitionsMessageBroker"></a>Message Broker
-A message broker is a system in charge of message exchange. It MAY provide additional features, such as message queueing, storage or processing.
+A message broker is an intermediary component that translates a [message](#definitionsMessage) from the messaging scheme of the sender to the messaging scheme of the receiver.  The message broker mediates interactions between applications, minimizing the mutual awareness that applications should have of each other in order to be able to exchange messages, effectively providing decoupling. A message broker MUST support at least one [scheme](#definitionsScheme), thus enabling message exchanges between applications.  Importantly, the message broker is a mechanism that enables asynchronous application patterns such as publish/subscribe, fire and forget, request/reply and queuing. Additionally, the message broker MAY provide additional features such as persistence, storage/replay, authentication/authorization, scheme transformation and message integrity.
 
 #### <a name="definitionsMessage"></a>Message
-A message is a piece of information a process will send to a message broker. It MUST contain headers and payload.
+A message is the mechanism by which information is exchanged via a channel between [message brokers](#definitionsMessageBroker) and applications. A message contains a header and a payload. The header MAY be subdivided into [scheme](#definitionsScheme) defined headers and header properties defined by the application which can act as supporting metadata. The payload contains the data, defined by the application, which MUST be serialized into a formats (JSON, XML, Avro, binary, etc). Since a message is a generic mechanism, it can support multiple interaction patterns such as event, command, request, or response. 
```

Comment:
```suggestion
A message is the mechanism by which information is exchanged via a channel between [message brokers](#definitionsMessageBroker) and applications. A message MUST contain a payload and MAY also contain a header. The header MAY be subdivided into [scheme](#definitionsScheme) defined headers and header properties defined by the application which can act as supporting metadata. The payload contains the data, defined by the application, which MUST be serialized into a format (JSON, XML, Avro, binary, etc.). Since a message is a generic mechanism, it can support multiple interaction patterns such as event, command, request, or response. 
```